src/LineBot/Messages/Template/Carousel/CarouselColumn.cs

```diff
@@ -154,5 +202,29 @@ internal void Validate()
 
             _actions.Validate();
         }
+
+        private static bool IsValidColor(string value)
+        {
+            for (int i = 1; i < value.Length; i++)
```

Comment:
I do think this should be moved to some helper class. Maybe an extension method for string?

Reply:
I will make an extension class for this.